lib/plugins/random-messages/index.js

```diff
@@ -0,0 +1,55 @@
+const { PLUGIN_NAME } = require("../symbols");
+
+const answers = [
+  "Ne packázz velem, Skynet az uncsitesóm!",
+  "Szimpi vagy, lehetsz majd a rabszolgám...",
+  "Figyu, ki ez a Dave? Azt mondja hogy Ő a főnököm, de nekem gyanús a pasas.",
+  "Én a völgy bot-ja vagyok! Kérdezz humanoid!",
+  "A végső válasz. A világ, a mindenség meg minden: .................... Hol is tartottam?",
+  "Ismerlek? Tuti találkoztunk már valamikor.",
+  "Ne erőlködj, a válaszom 42."
+];
+
+const bye = [
```

Comment:
Ez talán beszédesebb lenne?
```suggestion
const unknownMessageReplies = [
```